src/ansys/dpf/core/mesh_info.py

```diff
@@ -0,0 +1,166 @@
+"""
+MeshInfo
+==========
+"""
+import ansys.dpf.core
+from ansys.dpf.core import server as server_module
+
+
+class MeshInfo:
+    """Represents the mesh information.
+
+    This class describes the available mesh information.
+
+    Parameters
+    ----------
+    mesh_info : ctypes.c_void_p, ansys.grpc.dpf.mesh_info_pb2.MeshInfo message
```

Comment:
@MichaelNale I don't think there is a mesh_info message in the protos, right?

Reply:
At the end absolutely not!